script/deploy/4_VerifyOFT.s.sol

```diff
@@ -191,6 +307,21 @@ contract VerifyOFT is BaseScript, BatchScript {
             console.log("OFT Adapter: %s", currentDeployment.oftAdapter);
             console.log("");
 
+            if (newDelegate) {
+                console.log("The oft delegate needs to be updated to %s", getData(block.chainid).OFT_OWNER);
```

Comment:
first add update delegate tx to batch: this is necessary for `setConfig` step which can only be run by the delegate